velox/dwio/parquet/writer/Writer.h

```diff
@@ -33,7 +33,8 @@ struct ArrowContext;
 struct WriterOptions {
   bool enableDictionary = true;
   int64_t dataPageSize = 1'024 * 1'024;
-  int32_t rowsInRowGroup = 10'000;
+  int64_t rowsInRowGroup = 10'000;
```

Comment:
We should use [FlushPolicy](https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/FlushPolicy.h) to control this behavior to align with other formats.  We will receive data from metastore to set up flush behavior in the future (through something like a `FlushPolicyFactory`), so please keep the behavior aligned on reader level.  You can reuse a subclass of `FlushPolicy` here (can move some policies to common file if needed): https://github.com/facebookincubator/velox/blob/main/velox/dwio/dwrf/writer/FlushPolicy.h

Reply:
thank you @Yuhta for the reminder. I have updated it to follow the behavior of `dwrf/Writer`.